README.md

```diff
@@ -1,10 +1,14 @@
-# Spandx ![badge](https://github.com/mokhan/spandx/workflows/ci/badge.svg)
+![Spandx](logo.gif)
```

Comment:
We need to  put documentation here. http://spandx.github.io/  . I can do that but this projects read.md should be more technical. Like UML diagram or how to add plugin